src/main/java/org/tikv/txn/TxnKVClient.java

```diff
@@ -1,5 +1,27 @@
+/*
+ * Copyright 2019 The TiKV Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.tikv.txn;
 
+import org.tikv.common.TiSession;
+import org.tikv.txn.gc.GCWorker;
+
 public class TxnKVClient {
   // TODO: To be done.
+  public TxnKVClient(TiSession session) {
+    GCWorker worker = new GCWorker(session);
+    worker.run();
```

Comment:
Is the client designed to keep running for a long time?

Reply:
No it is not. Current GCWorker in TiDB is not designed for a single client (which is why I have to use a lock for multi-client scenario). In my opinion GCWorker should belong to an independent role rather than be moulded into computing layer. It should keep alive along with TiKV even if the client is not working. PD, for instance, should be a good role for starting GC requests but I understand it cannot by design.

Reply:
Yes! I always want to make GC an internal component of TiKV (and PD) so that users of pure KV won't need to write a client to GC the cluster anymore.
But here... I'm afraid this is not a very good idea if the programs that uses the TxnKVClient might exit soon...

Reply:
Yeah now that there isn't a good role to implement gc logic for client . Do you have any good ideas so that we can discuss its possibility? 

Here are a few potential choices: 

- start a GCWorker in background and make it it independent to our client (we probably call it GCClient)
- change our client into the same server-client structure as TiDB, and embed GC logic in the server with a much longer lifetime

The problem actually lies here:

GC is only valid when TiKV is in TXN mode. The GCWorker should be notified that it does not work with raw KV operations. Things get a bit difficult when trying to separate their logic from each other when they are concerned to be different components.

Reply:
I believe GC should be an internal functionality of TiKV, but it may need much time to discuss and implement it. Before that I think it's ok to adopt your first choice (which is also my idea).
As for raw kv.. It used to confuse some users that when they put some raw keys in TiKV of a TiDB cluster, TiDB got into failures soon. I think raw kv and txn kv shouldn't be used at the same time. So for clusters uses in raw kv mode,  we simply don't GC it.

Reply:
Fine, I will change this pr into an independent gc client before we have a better solution.